modules/vmfcore/java/com/intel/vmf/Metadata.java

```diff
@@ -0,0 +1,272 @@
+package com.intel.vmf;
+
+public class Metadata
+{
+    static
+	{
+        System.loadLibrary("vmf");
+	}
+	
+	public final long nativeObj;
+	
+	public static final long UNDEFINED_DURATION = 0;
+	
+	public static final long NUMOFFRAMES_DEFAULT = 1;
+	
+	protected Metadata (long addr)
+    {
+        if (addr == 0)
+            throw new java.lang.UnsupportedOperationException("Native object address is NULL");
+        
+        nativeObj = addr;
+    }
+	
+	public Metadata (MetadataDesc mdDescription)
+	{
+		nativeObj = n_Metadata (mdDescription.nativeObj);
+	}
+	
+	public Metadata (Metadata md)
+	{
+	    nativeObj = n_MetadataCopy (md.nativeObj);
+	}
+	
+	public void clear ()
+	{
+		//TO DO: implementation
```

Comment:
I suggest you throw an exception in not-yet-implemented methods, e.g.
`throw new java.lang.UnsupportedOperationException("Not Yet Implemented");`